examples/seq2seq/seq2seq_trainer.py

```diff
@@ -30,7 +30,6 @@
     "constant": get_constant_schedule,
     "constant_w_warmup": get_constant_schedule_with_warmup,
 }
-arg_to_scheduler_choices = sorted(arg_to_scheduler.keys())
```

Comment:
accident?

Reply:
It's not used in `seq2seq_trainer` -> it was only used in `Seq2SeqTrainingArguments`